extensions/wikia/JsonFormat/tests/JsonFormatTest.php

```diff
@@ -142,13 +142,22 @@ protected function checkClassStructure( $data, $structure ) {
 
 	protected function getParsedOutput( $wikitext ) {
 		return $this->memCacheDisabledSection( function () use ( $wikitext ) {
-			global $wgOut;
+			global $wgOut, $wgWikiaEnvironment;
 			$parser = ParserPool::get();
 			$htmlOutput = $parser->parse( $wikitext, new Title(), $wgOut->parserOptions() );
 
 			//check for empty result
 			if ( !empty( $wikitext ) ) {
 				$this->assertNotEmpty( $htmlOutput->getText(), 'Provided WikiText could not be parsed.' );
+				if( $wgWikiaEnvironment == WIKIA_ENV_DEV ) {
+					\Wikia\Logger\WikiaLogger::instance()->setDevModeWithES();
```

Comment:
Yes, it should be done in logger configuration.
However I do not want to change the logger just now, as this change is for debugging the root cause of the issue and will be removed once we know what happens.